src/test/common/urlUtils.test.ts

```diff
@@ -51,29 +51,38 @@ describe('urlUtils', () => {
     });
   });
 
+  const testUrlToRegex = (input: string, expectedRe: string) => {
+    const actualRe = urlToRegex(input);
+    expect(actualRe).to.equal(expectedRe);
+    expect(input).to.match(new RegExp(actualRe));
+  };
+
   describe('urlToRegex - case sensitive', () => {
     before(() => setCaseSensitivePaths(true));
     after(() => resetCaseSensitivePaths());
 
     it('works for a simple posix path', () => {
-      expect(urlToRegex('file:///a/b.js')).to.equal('file:\\/\\/\\/a\\/b\\.js|\\/a\\/b\\.js');
+      testUrlToRegex('file:///a/b.js', 'file:\\/\\/\\/a\\/b\\.js|\\/a\\/b\\.js');
     });
 
     it('works for a simple windows path', () => {
-      expect(urlToRegex('file:///c:/a/b.js')).to.equal(
-        'file:\\/\\/\\/[Cc]:\\/a\\/b\\.js|[Cc]:\\\\a\\\\b\\.js',
-      );
+      testUrlToRegex('file:///c:/a/b.js', 'file:\\/\\/\\/[Cc]:\\/a\\/b\\.js|[Cc]:\\\\a\\\\b\\.js');
     });
 
     it('works for a url', () => {
-      expect(urlToRegex('http://localhost:8080/a/b.js')).to.equal(
-        'http:\\/\\/localhost:8080\\/a\\/b\\.js',
-      );
+      testUrlToRegex('http://localhost:8080/a/b.js', 'http:\\/\\/localhost:8080\\/a\\/b\\.js');
     });
 
     it('space in path', () => {
-      expect(urlToRegex('file:///a/space%20path.js')).to.equal(
-        'file:\\/\\/\\/a\\/space( |%20)path\\.js|\\/a\\/space( |%20)path\\.js',
+      testUrlToRegex(
+        'file:///a/space%20path.js',
+        'file:\\/\\/\\/a\\/space(?: |%20)path\\.js|\\/a\\/space(?: |%20)path\\.js',
```

Comment:
I don't understand why it's correct to end up with a file URI that has a space in it, but I guess it was that way before

Reply:
Nor I. Maybe it is some weird thing in Electron or older Electron versions. I'll check again once we're at Electron 8 and see if the behavior is still present.